content/docs/validation.mdx

```diff
@@ -54,3 +54,21 @@ For example in React it could look like this:
 Here you can see the different modes in action
 
 <ValidationExample />
+
+## External Validation Errors
+
+External errors (e.g. coming from a backend) can be supplied to the form via the `additionalErrors` prop. These errors are mixed in with the regular errors coming from the AJV validation.
+The prop needs to be updated when necessary, as the [ValidationMode](#validationmode) has no effect on the external errors and only affects the errors coming from AJV. 
+
+```js
+const externalErrors: ErrorObject[] = ... ;
```

Comment:
Please mention that the `additionalErrors` should be stable or memoized to avoid unecessary rerendering. To also illustrate that we should extend the example a bit and for example wrap the `<JsonForms>` component into a proper component.